wp-rocket-cache-feed/wp-rocket-cache-feed/wp-rocket-cache-feed.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Plugin Name: WP Rocket Cache Feed
+ * Plugin URI: https://wp-rocket.me
+ * Description: Activate caching of WordPress RSS feeds by WP Rocket
+ * Version: 1.0
+ * Author: WP Media
+ * Contributors: Remy Perona
+ * Author URI: http://wp-media.me
+ * Licence: GPLv2
+ *
+ * Text Domain: wp-rocket-cache-feed
+ * Domain Path: languages
+ *
+ * Copyright 2017 WP Rocket
+ * */
+ 
+defined( 'ABSPATH' ) or die( 'Cheatin&#8217; uh?' );
+
+require( plugin_dir_path( __FILE__ ) . '/inc/functions.php' );
+
+/**
+ * Run when plugin is loaded
+ *
+ * @since 1.0
+ */
+function wp_rocket_cache_feed_init() {
+	if ( ! is_wp_rocket_active() ) {
+		return;
```

Comment:
Let’s get in the habit of always returning an actual value for [tests](https://tfrommen.de/an-introduction-to-unit-testing-for-wordpress/):

```php
return false;
```